src/main/java/slimeknights/mantle/command/client/BookCommand.java

```diff
@@ -186,6 +192,15 @@ private static int doExportImages(ResourceLocation book, int scale) {
             Mantle.logger.error("Failed to save screenshot", e);
             throw new CommandRuntimeException(Component.translatable(EXPORT_FAIL));
           }
+
+          if (page >= 0) {
+            File file = Paths.get(screenshotDir.toString(), "page_" + page + ".html").toFile();
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
+              writer.write(screen.toHTML());
+            } catch (IOException ignored) {
+              // TODO: handle exception
+            }
+          }
```

Comment:
This should be a separate command. One for exporting to images, one for exporting to HTML. We don't want one breaking preventing us from using the other (and the image export command has broken every update in some small way)